src/rpc/blockchain.cpp

```diff
@@ -604,6 +604,20 @@ static UniValue SoftForkDesc(const std::string &name, int version, CBlockIndex*
     return rv;
 }
 
+static UniValue BIP9SoftForkDesc(const std::string& name, const Consensus::Params& consensusParams, Consensus::DeploymentPos id)
```

Comment:
Mhmm, I can only find one instance of the string "BIP9SoftForkDesc" in the changes.

re itarate with knowledge of the names, what's wrong with 
```
for (int i = 0; i < MAX_VERSION_BITS_DEPLOYMENTS; ++i)
``` ?